standard/standard_9.php

```diff
@@ -36,27 +40,35 @@ function array_merge_recursive(#[PhpStormStubsElementAvailable(from: '5.3', to:
  * @param array $array <p>
  * The array in which elements are replaced.
  * </p>
- * @param array ...$replacements [optional] <p>
+ * @param array ...$replacements <p>
  * The array from which elements will be extracted.
  * </p>
  * @return array or null if an error occurs.
  */
 #[Pure]
-function array_replace(array $array, array ...$replacements): array {}
+function array_replace(
+    array $array,
+    #[PhpStormStubsElementAvailable(from: '5.3', to: '7.0')] $arrays,
+    #[PhpStormStubsElementAvailable(from: '7.1')] array ...$replacements
```

Comment:
same stuff here: 3rd parameter named `...` for 5.4-5.5 PHP versions: https://3v4l.org/31rEt#v540

Reply:
Answer it the same as above